cmd/soroban-rpc/docker/k8s_deployment.yml

```diff
@@ -0,0 +1,128 @@
+# running captive core with on-disk mode limits RAM to around 2G usage, but 
+# requires some dedicated disk storage space that has at least 3k IOPS for read/write.
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: soroban-rpc-futurenet-core-storage
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Gi
+  storageClassName: default
+  volumeMode: Filesystem  
+---   
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    fluxcd.io/ignore: "true"
+  labels:
+    app: soroban-rpc-futurenet
+  name: soroban-rpc-futurenet
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: soroban-rpc-futurenet
+  template:
+    metadata:
+      annotations:
+        fluxcd.io/ignore: "true"
+      creationTimestamp: null
+      labels:
+        app: soroban-rpc-futurenet
+    spec:
+      initContainers:
+      - name: clean-volume
+        image: busybox:1.28
+        command: ["rm", "-rf", "/opt/stellar/soroban-rpc/captive-core"]
+        volumeMounts:
+        - mountPath: /opt/stellar/soroban-rpc
+          name: captive-core-data
+      containers:    
+      - name: rpc-server
+        image: stellar/quickstart:soroban-dev@sha256:81c23da078c90d0ba220f8fc93414d0ea44608adc616988930529c58df278739
+        args: ["--futurenet", "--enable-soroban-rpc", "--logs"]
```

Comment:
This is meant to be separate from the soroban-rpc helm chart yes?

Reply:
yes, this manifest is intended as short-term gap to provide deployment up for now. Next up is to create formal deployment artifact for rpc on https://github.com/stellar/soroban-tools/issues/382, that ticket addresses creation of a dedicated dockerfile definition for rpc runtime image(includes captive core config) and integrate that to the helm chart.